web/static/js/socket.js

```diff
@@ -69,19 +71,22 @@ $chatInput.on("keypress", event => {
 })
 
 channel.on("new_msg", payload => {
-  $messagesContainer.prepend(`<li>[${Date()}] ${payload.from}: ${payload.body}</li>`)
+  displayMessage(payload)
 })
 
-priv.on("whisper", payload => {
-  $messagesContainer.prepend(`<li>[${Date()}] ${payload.from}: ${payload.body}</li>`)
+// Really broad event - captures more than we want
+channel.on("phx_reply", payload => {
+  let response = payload.response
+  // Hacky way to narrow down to the replies we're interested in
+  if(payload.status === "ok" && response && response.kind === "private" && Array.isArray(response.messages)) {
+    response.messages.forEach(body => {
+      displayMessage({from: response.from, body: body})
+    })
+  }
```

Comment:
Ha - yeah, I just learned that from asking on the mailing list. Updated the code accordingly.